contributor-docs/adrs/adr-002-behavior-isolation.md

```diff
@@ -2,9 +2,13 @@
 
 ## Status
 
-~~Approved 2021-02-04~~
 Superseded by [ADR 010](./adr-010-behavior-isolation.md) (2023-01-31)
 
+| Stage    | Status |
+| -------- | ------ |
+| Approved | ✅     |
+| Adopted  | ✅     |
+
```

Comment:
Presuming that these are here for at-a-glance understanding of the progress of these ADRs, and given this one is now superseded, does it make sense to add a row or otherwise turn these emojis into :warning: or something?

Reply:
+1 to Keith. If an ADR is superseded, I might suggest removing both "Approved" and "Adopted" statuses and introducing "Superseded" with a link to the superseding ADR.